runtime/frequency/src/lib.rs

```diff
@@ -493,6 +496,9 @@ impl pallet_capacity::Config for Runtime {
 	type StakingRewardsPastErasMax = ConstU32<30u32>;
 	type RewardsProvider = Capacity;
 	type MaxRetargetsPerRewardEra = ConstU32<16>;
+	// Value determined by desired inflation rate limits for chosen economic model
+	type RewardPoolEachEra = ConstU128<{ currency::CENTS.saturating_mul(172_602_740u128) }>;
```

Comment:
Question: Where does this magic number come from?

Reply:
Spreadsheet referenced above - I'm not sure it's okay to share in a public repo.  DM me for the link.

Reply:
It is ~4.5% annually of the initial token mint (1 billion).

`14/365 * 1000000000 * 0.045 = 1726027`

@shannonwells would it be worth putting in the calculation directly? Either way, we can comment it as such that in this or a future pr.